qiskit_nature/problems/sampling/protein_folding/peptide/chains/main_chain.py

```diff
@@ -105,57 +92,30 @@ def _build_main_chain(
                 side_chain,
             )
             main_chain.append(main_bead)
-        main_bead = MainBead(main_chain_len - 1, None, None, None)
+        main_bead = MainBead(main_chain_len - 1, "", None, None)
         main_chain.append(main_bead)
         return main_chain
 
     @staticmethod
     def _validate_side_chain_lengths(
         main_chain_len: int,
-        side_chain_lens: List[int],
-        side_chain_residue_sequences: List[Optional[str]],
+        side_chain_residue_sequences: List[str],
     ) -> None:
-        if side_chain_lens is not None and main_chain_len != len(side_chain_lens):
-            raise InvalidSizeException(
-                f"The length of list of side chain lengths provided: {len(side_chain_lens)}, "
-                f"does not equal the length of the main chain: {main_chain_len}"
-            )
-        if side_chain_lens is not None and main_chain_len != len(side_chain_residue_sequences):
+        if main_chain_len != len(side_chain_residue_sequences):
             raise InvalidSizeException(
                 f"The length of list of main chain residue sequences: "
```

Comment:
```suggestion
        if main_chain_len != len(side_chain_residue_sequences):
            raise InvalidSizeException(
                f"The length of list of side chain residue sequences: "
```
I think the message should say `side` chain.... not `main` chain right